client/state/reader-ui/sidebar/actions.js

```diff
@@ -44,3 +45,12 @@ export function toggleReaderSidebarFollowing() {
 		type: READER_SIDEBAR_FOLLOWING_TOGGLE,
 	};
 }
+
+export function toggleReaderSidebarRecent() {
+	recordAction( 'toggled_reader_sidebar_followed_sites' );
+	recordGaEvent( 'Toggled Reader Sidebar Followed Sites' );
+	recordTrack( 'calypso_reader_sidebar_followed_sites_toggled' );
```

Comment:
I noticed some of the other actions are using present tense in the arguments and wondered if we should stick to the same convention here.

Reply:
Thanks for noticing these inconsistencies! When looking at them I realized I don't need to add state management, because it already exists, it just wasn't being used. Updated in https://github.com/Automattic/wp-calypso/pull/95689/commits/5948150c5125ca3009c006d0a9b74cf6021561f0

Reply:
Ah nice catch! I totally looked over that.